src/TracerState.h

```diff
@@ -89,6 +90,18 @@ class TracerState {
             binary_,
             compression_level_);
 
+        dynamic_function_definitions_data_table_ = dynalyzer_create_data_table(
```

Comment:
We don't want multiple function definition tables. This leads to increase in data size.
We can always query functions by id from the `function_definitions` table.